chromaui/chromatic-e2e
# Support using `httpCredentials` in Playwright global config for basic authentication

Issue: #AP-4955

## What Changed



The Playwright test fixture will now respond to basic authentication challenges using the `httpCredentials` supplied in the global config, if available.

## How to test



An [internal e2e test](https://github.com/chromaui/chromatic-e2e/blob/960f2b9d17de661bcedc12b417b167316d1e9f87/packages/playwright/tests/auth.spec.ts) was created to validate this, but you can create you own like it:

1. Create an application that requires basic authentication for at least one page
2. In the Playwright global config that tests that application, set `httpCredentials`
3. Create a test that accesses that secure page and verify that it passes
